www/stos02/sem_web/Apple_Home.html

```diff
@@ -0,0 +1,109 @@
+
+                snad se neurazíš, že ti po všech těch letech co to spolu táhneme budu tykat. Je tomu šest let
+                co jsme
+                ti psal naposledy a ty si jistě na můj Dopis pro Apple ještě vzpomínáš. Tentokrát mám však na
+                srdci
+                něco jiného než před lety, něco pro mě osobně mnohem důležitějšího. Víš zvykl jsem si na to, že
+                si rok,
+                co rok budu kupovat nový iPhone a je to zkrátka koloběh, který mi vůbec nevadí. Zvykl jsme si
+                že lidé
+                okolo mě rok co rok mění své Apple Watch a pořád se smějí tomu, že nosím Garminy. Vůbec mi to
+                nevadí a
+                chápu že z toho máš miliardové zisky, které nutně potřebuješ. Poslední dobou ...
+            </div>
+        </div>
+
+
+        <div class="article__text">
+            <div class="article__title">
+                <a href="news.html">Apple oficiálně potvrdil příchod čtyř hlavních novinek do iOS 12.1</a>
+            </div>
+            <div class="article_description">V krabičkách iPhonů bývá jen zkrácená verze manuálu k jeho
+                ovládání či
+                pro vysvětlení některých funkcí systému. Plná verze manuálu je však volně dostupná v
+                elektronické
+                podobě v aplikaci Knihy.
+                Právě tam se objevil před více než měsícem i manuál pro iPhony s iOS 12, který obsahuje
+                popis
+                veškerých novinek a funkcí systému. Ten se Apple rozhodl včera aktualizovat o pár
+                detailů, díky...
+            </div>
+        </div>
+    </div>
+    <footer>
+        <div class="footer">
+            <div class="right">
+                <h2> LETEM SVĚTEM APPLEM</h2>
+                <p>
+
+                    © Všechna práva vyhrazena Text Factory s. r. o. © 2011 - 2018. Design vytvořil JW, zdrojový kód
+                    Erik
+                    Bročko.
+                    Poučení o ochraně osobních údajů naleznete zde.
+                    Podmínky používání médií patřících do skupiny Text Factory s. r. o. naleznete přímo zde. Informace
+                    o
+                    souborech
+                    Cookies, které využíváme, naleznete přímo zde.
+
+                    iPouzdro.cz - kryty na iPhone a obaly na iPad | Text Factory s. r. o. | Rychlost internetu |
+                    Jablíčkář.cz -
+                    Apple novinky | SamsungMagazine.eu - Novinky o Samsungu | Webhosting, hosting, domény | Chytré
+                    držáky
+                    mobilů na
+                    kolo Quad Lock a FINN zakoupíte v eshopu Kábelmanie.cz | Pouzdro pro iPhone z kůže
+
+                </p>
+            </div>
+
+            <div class="social">
+                <div class="son_twitter">
+                    <a href="https://twitter.com/lsaeu" target="twitter_blank"><img src="images/Twitter.png" alt="twitter"></a></div>
```

Comment:
jak funguje neco a pak _blank? napr twitter_blank?

Reply:
mělo by to být k tomu, aby se stránka načetla v novém okně

Reply:
jo, ale to ma byt jen "_blank"

Reply:
jasný, chápu. Děkuju